### PR TITLE
added translations

### PR DESCRIPTION
**Pull request by Buhle79**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 rasa==1.10.2
 protobuf==3.19.4
+importlib-metadata==4.13.0
```
Comment (bearnard) [inline requirements.txt:3]: Why is this added?

**Comment by fritzbrand** [comment]
👍

File: requirements.txt (hunk 1 above)
Comment (Buhle79) [reply requirements.txt:3]: Because importlib-metadata releases v5.0.0  which it remove deprecated endpoint. We get error AttributeError: **'EntryPoints' object has no attribute 'get'**. it's either we upgrade python or downgrade this library. @bearnard

**Comment by bearnard** [comment]
👍
